src/metabase/models/permissions_group.clj

```diff
@@ -109,17 +111,20 @@
 (defn ^:hydrate members
   "Return `Users` that belong to GROUP-OR-ID, ordered by their name (case-insensitive)."
   [group-or-id]
-  (db/query {:select    [:user.first_name
-                         :user.last_name
-                         :user.email
-                         [:user.id :user_id]
-                         [:pgm.id :membership_id]]
-             :from      [[:core_user :user]]
-             :left-join [[:permissions_group_membership :pgm] [:= :user.id :pgm.user_id]]
-             :where     [:and [:= :user.is_active true]
-                              [:= :pgm.group_id (u/the-id group-or-id)]]
-             :order-by  [[:%lower.user.first_name :asc]
-                         [:%lower.user.last_name :asc]]}))
+  (db/query (cond-> {:select    [:user.first_name
+                                 :user.last_name
+                                 :user.email
+                                 [:user.id :user_id]
+                                 [:pgm.id :membership_id]]
+                     :from      [[:core_user :user]]
+                     :left-join [[:permissions_group_membership :pgm] [:= :user.id :pgm.user_id]]
+                     :where     [:and [:= :user.is_active true]
+                                 [:= :pgm.group_id (u/the-id group-or-id)]]
+                     :order-by  [[:%lower.user.first_name :asc]
+                                 [:%lower.user.last_name :asc]]}
+
+              (premium-features/enable-advanced-permissions?)
```

Comment:
when `advanced-permissions` is enabled add `is_group_manager` when hydrate `:members`